spicy/toolchain/src/compiler/parser/parser.yy

```diff
@@ -904,6 +904,9 @@ expr_g        : '(' expr ')'                     { $$ = hilti::expression::Group
                                                    auto index = hilti::expression::Ctor(hilti::ctor::UnsignedInteger($1, 64, __loc__), __loc__);
                                                    $$ = hilti::expression::UnresolvedOperator(hilti::operator_::Kind::Index, {std::move(captures), std::move(index)}, __loc__);
                                                  }
+              | expr_h
```

Comment:
Hmm ... I think this should have a priority that's either like normal function calls (add to `expr_d`) or other ctors (add to `expr_f`); not quite sure which one, but I'd say the latter.